openpype/hosts/max/plugins/publish/validate_renderpasses.py

```diff
@@ -0,0 +1,156 @@
+                report, title="Invalid Render Passes")
+
+    @classmethod
+    def get_invalid(cls, instance):
+        """Function to get invalid beauty render outputs and
+        render elements.
+
+        1. Checking Render Output Folder matches the name of
+        the current Max Scene
+        e.g. The name of the current Max scene:
+                John_Doe.max
+            The expected render output directory:
+            {root[work]}/{project[name]}/{hierarchy}
+            /{asset}/work/{task[name]}/render/3dsmax/John_Doe/
+
+        2. Checking the image extension(s) of the render output(s)
+          does not match with image format in OP/AYON setting.
+        e.g. The current image format in the setting: png
+            The expected render outputs: Joe_Doe.png
+
+        3. Checking the filename of render element is not ended
+            with the name of render element from the 3dsMax Render
+            Element Manager.
+        e.g. The name of render element: RsCryptomatte
+            The expected filename: {InstanceName}_RsCryptomatte.png
+
+        Args:
+            instance (pyblish.api.Instance): instance
+            filename (str): filename of the Max scene
+
+        Returns:
+            list: list of invalid filename which doesn't match
+                with the project name
+        """
+        invalid = []
+        file = rt.maxFileName
+        filename, ext = os.path.splitext(file)
+        if filename not in rt.rendOutputFilename:
+            cls.log.error(
+                "Render output folder "
+                "doesn't match the max scene name! "
+            )
+            invalid_folder_name = os.path.dirname(
+                rt.rendOutputFilename).replace(
+                    "\\", "/").split("/")[-1]
+            invalid.append(("Invalid Render Output Folder",
+                            invalid_folder_name))
+        beauty_fname = os.path.basename(rt.rendOutputFilename)
+        ext = os.path.splitext(beauty_fname)[-1].lstrip(".")
+        invalid_image_format = cls.get_invalid_image_format(
+            cls, instance, ext)
+        invalid.extend(invalid_image_format)
+        renderer_class = get_current_renderer()
+        renderer = str(renderer_class).split(":")[0]
+        if renderer in [
+            "ART_Renderer",
+            "Redshift_Renderer",
+            "V_Ray_6_Hotfix_3",
+            "V_Ray_GPU_6_Hotfix_3",
+            "Default_Scanline_Renderer",
+            "Quicksilver_Hardware_Renderer",
+        ]:
+            render_elem = rt.maxOps.GetCurRenderElementMgr()
+            render_elem_num = render_elem.NumRenderElements()
+            for i in range(render_elem_num):
+                renderlayer_name = render_elem.GetRenderElement(i)
+                renderpass = str(renderlayer_name).split(":")[-1]
+                rend_file = render_elem.GetRenderElementFilename(i)
+                if not rend_file:
+                    cls.log.error(f"No filepath for {renderpass}")
```

Comment:
Should this mention that it's about a render element? Or is that clear solely from the `renderpass` string?
```suggestion
                    cls.log.error(f"No filepath for render element {renderpass}")
```